stable/nextcloud/README.md

```diff
@@ -150,6 +150,14 @@ The following table lists the configurable parameters of the nextcloud chart and
 | `hpa.maxPods`                                                | Max. pods for the Nextcloud HorizontalPodAutoscaler     | `10`                                        |
 | `deploymentAnnotations`                                      | Annotations to be added at 'deployment' level           | not set                                     |
 | `podAnnotations`                                             | Annotations to be added at 'pod' level                  | not set                                     |
+| `metrics.enabled`                                            | Start a single Prometheus metrics exporter              | `false`                                     |
+| `metrics.https`                                              | Wether https is used for scraping Nextcloud metrics     | `false`                                     |
+| `metrics.timeout`                                            | When the scrape times out                               | `5s`                                        |
+| `metrics.image.repository`                                   | Nextcloud metrics exporter image name                   | `xperimental/nextcloud-exporter`            |
+| `metrics.image.tag`                                          | Nextcloud metrics exporter image tag                    | `latest`                                    |
+| `metrics.image.pullPolicy`                                   | Nextcloud metrics exporter image pull policy            | `IfNotPresent`                              |
+| `metrics.podAnnotations`                                     | Additional annotations for metrics exporter             | `{prometheus.io/scrape: "true", prometheus.io/port: "9205"}` |
```

Comment:
these would probably then be rather annotations for the service (if you expose via a service)

Reply:
I've moved the annotations to the service object.